src/pages/patientView/copyNumberAlterations/CopyNumberTableWrapper.tsx

```diff
@@ -57,6 +58,12 @@ export default class CopyNumberTableWrapper extends React.Component<ICopyNumberT
         enableCivic: false
     };
 
+    getCytoband(hugoGeneSymbol:string) {
+        const refGene =  _.find(this.props.refGenes,(gene:ReferenceGenomeGene)=>
+            gene.hugoGeneSymbol===hugoGeneSymbol);
+        return refGene? refGene.cytoband: "-";
```

Comment:
returing "-" seems like a presentation choice.  I would return null or undefined and then have view code specify what to show in that case.

Reply:
agree.

Reply:
replaced "-" with undefined.